mavros/launch/px4_config.yaml

```diff
@@ -44,20 +44,20 @@ cmd:
 
 # global_position
 global_position:
-  frame_id: "map"             # origin frame
-  child_frame_id: "base_link" # body-fixed frame
+  frame_id: "local_origin"  # origin frame
+  child_frame_id: "fcu_utm" # body-fixed frame
   rot_covariance: 99999.0   # covariance for attitude?
   gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
   use_relative_alt: true    # use relative altitude for local coordinates
   tf:
     send: false               # send TF?
-    frame_id: "map"  # TF frame_id
+    frame_id: "local_origin"  # TF frame_id
     global_frame_id: "earth"  # TF earth frame_id
-    child_frame_id: "base_link" # TF child_frame_id
+    child_frame_id: "fcu_utm" # TF child_frame_id
 
 # imu_pub
 imu:
-  frame_id: "base_link"
+  frame_id: "fcu"
```

Comment:
Why are you changing this frames? We should use the frame naming according to [ROS REP 105](https://www.ros.org/reps/rep-0105.html).

Reply:
Ok, I can change it back. Should we keep the frame_id that are not related to the tf tree as they were? But then they don't match the tf tree anymore.

Reply:
Let's build the TF tree in a way that we are sure that the naming matches the ROS REP 105. Everything that goes out of the "normal" frame convention should at least have the REP 105 frame naming as a prefix. Ex: `base_link` and `base_link_frd` or `odom` and `odom_ned`.